torch/distributed/checkpoint/state_dict_loader.py

```diff
@@ -283,12 +285,26 @@ def _load_state_dict_from_keys(
         "torch.distributed.checkpoint._load_state_dict_from_keys"
     )
 
+    no_dist = not (dist.is_available() and dist.is_initialized())
+    if no_dist:
+        warnings.warn(
+            "torch.distributed is unavailable or uninitialized, assuming the intent is to load in a single process."
```

Comment:
Mind setting a stack level for this warning?

Reply:
Hey @Skylion007 thanks for the suggestion, I'll see if I can circle back to this in a follow-up and update all of our warnings.